Pipeline/PackageBuildOutputs/README.md

```diff
@@ -24,6 +24,9 @@ This section provides a more detailed explanation of how the PackageBuildOutputs
       1. Parse and extract build output information for records of type *ExecuteRecord* and *CopyToPDSRecord*. (Requires at least DBB 1.0.8.)
       1. Remove output entries that have no `deployType` set and remove unwanted outputs such as outputs with the `deployType` equal to `ZUNIT-TESTCASE`.
    1. If processing multiple build reports, a cumulative hashmap of output records is created to be able to combine outputs from multiple pipeline builds into a single tar file.
+   	  1. The key of the map, used in the calculation of the artifacts to be deployed, is the combination of the member name and the deploy type.
```

Comment:
Can you please add a comment, that the script is generally not managing deletions? Something we missed to call out earlier.

Reply:
Added a sentence about it.